src/App.js

```diff
@@ -69,13 +94,17 @@ class App extends Component {
     return (
       <div className="App">
         <header className="App-header">
-            Star Wars goodness 
+            * Star Wars Goodness *
             <SearchBar
              inputElement= {this.inputElement}
              onChangeInput = {this.handleSearchInput}
              currentItem={this.state.searchedItem}
              addItemForSearch={this.addItemForSearch} 
               />
+            <SearchBarDetails 
+            searchedItemDetails={this.state.searchedItemDetails.text}
+            showSearchBarDetails={this.state.foundDetails}
```

Comment:
It's good to destructure this within the render method. Anything that requires you to repeat accessing properties on the same object more than once can be hoisted up:

```js
 const = searchedItemDetails, foundDetails = this.state
```